docs/website/docs/dlt-ecosystem/staging.md

```diff
@@ -96,4 +100,21 @@ In essence, you need to set up two destinations and then pass them to `dlt.pipel
 
     Run the pipeline script as usual.
 
-> 💡 Please note that `dlt` does not delete loaded files from the staging storage after the load is complete.
+:::tip
```

Comment:
I'd run this whole new section through chatgpt to grammar correct (or run the grammar checker on this page)